python_modules/dagster/dagster/_core/execution/plan/inputs.py

```diff
@@ -186,7 +186,10 @@ def load_input_object(
 
         yield from _load_input_with_input_manager(loader, load_input_context)
 
-        metadata = load_input_context.consume_metadata()
+        metadata = {
```

Comment:
previously, `consume_metadata` returned definition and execution-time metadata as a single dict. It now just returns the execution-time metadata for consistency with `OutputContext` so we need to get the definition metadata as well